documentation/modules/ROOT/pages/04-quarkus-extensions.adoc

```diff
@@ -770,7 +770,7 @@ Open `pass:[$PROJECT_HOME]/src/main/java/java/com/example/FruitResource.java` an
 [#quarkussec-jwt-claim-username]
 [source,config,subs="+macros"]
 ----
-@Claim(standard = Claims.preferred_username)
+@Inject @Claim(standard = Claims.preferred_username)
```

Comment:
In previous versions `@Inject` was not necessary, in fact in Quarkus we are trying to avoid the inject annotation in several cases.

Reply:
I know but there must be a regression. At least it was not working for me when I tried yesterday. And the JWT guide uses @Inject. We should open a side issue on Quarkus proper.

Reply:
https://github.com/quarkusio/quarkus/issues/5160